CMakeLists.txt

```diff
@@ -163,4 +99,58 @@ install(FILES
     DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Arcus
 )
 
-include(CPackConfig.cmake)
+# Create the Python bindings
+if(BUILD_PYTHON)
+    list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
+
+    if(NOT DEFINED Python_VERSION)
+        set(Python_VERSION
+                3.10
+                CACHE STRING "Python Version" FORCE)
+        message(STATUS "Setting Python version to ${Python_VERSION}. Set Python_VERSION if you want to compile against an other version.")
+    endif()
+    if(APPLE)
+        set(Python_FIND_FRAMEWORK NEVER)
+    endif()
+    find_package(cpython ${Python_VERSION} QUIET COMPONENTS Interpreter Development)
+    if(NOT TARGET cpython::cpython)
+        find_package(Python ${Python_VERSION} EXACT REQUIRED COMPONENTS Interpreter Development)
+    else()
+        add_library(Python::Python ALIAS cpython::python)
+        set(Python_SITEARCH "${CMAKE_INSTALL_PREFIX}/lib/python3.10/site-packages")
+        set(Python_EXECUTABLE ${cpython_PACKAGE_FOLDER_RELEASE}/bin/python3)
+        set(ENV{PYTHONPATH} ${Python_SITEARCH})
+    endif()
+    message(STATUS "Linking and building ${project_name} against Python ${Python_VERSION}")
+
+    find_package(SIP REQUIRED 6.5.0)
+
+    add_library(pyArcus INTERFACE ${CMAKE_SOURCE_DIR}/python/PythonMessage.cpp)
+    set_project_standards(pyArcus)
+    set_rpath(TARGETS
+                pyArcus
+              PATHS
+                "$<$<PLATFORM_ID:Linux>:usr/bin>"
+                "$<$<PLATFORM_ID:Linux>:usr/bin/lib>"
+                "$<$<PLATFORM_ID:Darwin>:../lib>"
+                "$<$<PLATFORM_ID:Darwin>:../Resources/lib/>"
+                "../../"
+              RELATIVE)
+    use_threads(pyArcus)
+
+    target_include_directories(pyArcus
+            INTERFACE
+            $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+            $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/python/>
+            )
+
+    target_compile_features(pyArcus INTERFACE cxx_std_17)
```

Comment:
I think this is defined by the standard project options already, right? It sets `CMAKE_CXX_STANDARD` to 17 at least.
```suggestion

```